src/NorthwindWebsite/API/Controllers/CategoriesController.cs

```diff
@@ -23,4 +26,41 @@ public async Task<ActionResult<List<Category>>> GetCategories()
 
         return categories.ToList();
     }
+
+    [HttpGet("{categoryId}/image")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
```

Comment:
For other types it is also a good criteria to specify exact return type, like `typeof(string)`. Swagger will automatically suggest an example of returned type for this case. If you return nothing, you can leave `typeof(void)` (hope it will not throw any exceptions now on performing request sending, in recent versions of ASP.Net Core it was occurring)